fuel-coroutines/build.gradle.kts

```diff
@@ -1,7 +1,7 @@
 dependencies {
     api(project(Fuel.name))
 
-    api(KotlinX.Coroutines.jvm)
+    api(Kotlinx.Coroutines.jvm)
```

Comment:
this change was needed because the plugin embed two top-level objects with the name `AndroidX` and `KotlinX`, that clash with the two same objects defined in the buildSrc